content/docs/getting-started/asyncapi-documents.md

```diff
@@ -40,6 +40,6 @@ The AsyncAPI document is a machine-readable definition of your event-driven API.
 
 Your API documentation is now machine-readable –easily parseable by code— so the myriad of useful applications is endless.
 
-Enough speaking, let's get our hands dirty with some examples. Learn how to create an AsyncAPI document defining a "Hello world" application.
+Enough speaking, let's get your hands dirty with some examples. Learn how to create an AsyncAPI document defining a "Hello world" application.
```

Comment:
"Let's get **your** hands dirty" sounds weird. Doesn't it?

Reply:
Sounds weird only if you have dirty thoughts on Valentine's Day 😆 
It is a completely normal sentence https://www.ldoceonline.com/dictionary/get-your-hands-dirty. It is a pretty informal sentence, but I think we do not have to be super formal in tech docs.

Reply:
Oh yeah, I'm not concerned about "get your hands dirty". It's more about the "Let's .... **your**...". You know what I mean? It's ok anyway 👍